configs/config.dev.yaml

```diff
@@ -67,13 +68,13 @@ user_session:
   sign_key: '123'
 
 supported_timezones:
+  - 'Asia/Kolkata'
```

Comment:
small nit: can you change this ordering back to how it was previously so timezones are grouped by regions in the list

Reply:
changes done